src/Common/src/System/Diagnostics/Debug.Windows.cs

```diff
@@ -75,6 +77,25 @@ private static void WriteToDebugger(string message)
                     Interop.mincore.OutputDebugString(message ?? string.Empty);
                 }
             }
+
+            private static unsafe bool WriteToStdErr(string message)
+            {
+                IntPtr handle = Interop.mincore.GetStdHandle(Interop.mincore.HandleTypes.STD_ERROR_HANDLE);
```

Comment:
Does this do the right thing when you don't have a console associated with your process?